internal/boxcli/run.go

```diff
@@ -41,6 +43,30 @@ func RunCmd() *cobra.Command {
 	return command
 }
 
+func InstallCmd() *cobra.Command {
+	flags := runCmdFlags{}
+	command := &cobra.Command{
+		Use:   "install",
+		Short: "Installs all packages mentioned in devbox.json",
+		Long: "Starts a new shell and installs all packages mentioned in devbox.json or a json file" +
+			"specified via --config. \n\n Then exits the shell when packages are done installing.\n\n ",
+		Args:    cobra.MaximumNArgs(0),
+		PreRunE: ensureNixInstalled,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := runScriptCmd(cmd, []string{":"}, flags)
```

Comment:
for shell n00bs like myself, would you mind adding a small comment explaining that `:` is a noop in shell?